STM32/DC/Core/Src/DCBoard.c

```diff
@@ -383,4 +419,5 @@ void DCBoardLoop(const char* bootMsg)
     // Turn off pins if they have run for requested time
     autoOff();
     checkButtonPress();
+    handlePorts();
```

Comment:
I'm not quite sure whether this could pose a problem or not, but given it is an actuation board I think we should be careful with running handlePorts if there is a SW mismatch. 

The timers are started in main.c and initialised to a safe state, but if for whatever reason a command is sent to them they could begin to actuate even if there is a sw mismatch where the behaviour is unknown.

One way to tackle this would be to move the initialisation of the timers to the DCBoardInit function and then only initialise them if there is no sw mismatch.

Reply:
Added an if statement to prevent this